tpkg/ixfr_retry.tdir/ixfr_retry.test

```diff
@@ -0,0 +1,62 @@
+# #-- ixfr_retry.test --#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# use .tpkg.var.test for in test variable passing
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+. ../common.sh
+
+# ixfr_retry uses two zones bar.foo. and baz.foo. NSD consistently issues IXFR
+# requests for baz.foo. before bar.foo. This ensures that changes are
+# processed in the same order. This behavior is used to the advantage of this
+# test because the initial reload must discard the set of changes for the
+# second zone to verify whether the set of changes for the first zone are kept
+# around.
+
+NSD="../../nsd"
+
+#valgrind $NSD -c nsd.conf -L 5 -F 0xFFFF -p $NSD_PORT
+$NSD -c nsd.conf -L 5 -F 0xFFFF -p $NSD_PORT
```

Comment:
Because this test needs the debug output, you need to test that debug is enabled, and skip the test.  With like `if $NSD -h 2>&1 | grep "L level"; then echo "debug enabled, okay"; else echo "debug output not available, skip test"; exit 0; fi`